shared/js/content-scripts/settings-toggle.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
This content script is only injected into the post install page. It does two things there:
1) Connects the 'activate tracker blocking' buttons on the page to the extension, so that clicking them changes the `trackerBlockingEnabled` setting.
2) Listens for a specific message from the background page when tracker blocking is enabled, then forwards the message to the page so it can remember that tracker blocking was enabled.